subprojects/persistent-cache/src/main/java/org/gradle/cache/internal/DefaultPersistentDirectoryCache.java

```diff
@@ -37,7 +38,7 @@
 import java.util.concurrent.TimeUnit;
 
 public class DefaultPersistentDirectoryCache extends DefaultPersistentDirectoryStore implements ReferencablePersistentCache {
-    public static final int CLEANUP_INTERVAL = 7;
+    public static final int CLEANUP_INTERVAL_IN_HOURS = 24;
```

Comment:
The change from days to hours is only because the debug message makes more sense?

Reply:
Yes.